src/HeadlessUniversalCheckout/Managers/VaultManager.ts

```diff
@@ -50,20 +51,21 @@ class PrimerHeadlessUniversalCheckoutVaultManager {
                 console.error(err);
                 reject(err);
             }
-        })
+        });
     }
 
-    async validate(vaultedPaymentMethodId: String, additionalData: PrimerVaultedPaymentMethodAdditionalData): Promise<PrimerValidationError[]> {
+    async validate(vaultedPaymentMethodId: String, additionalData: VaultedPaymentMethodAdditionalData): Promise<ValidationError[]> {
         return new Promise(async (resolve, reject) => {
             try {
-                const errors: PrimerValidationError[] =
+                const data =
```

Comment:
Same as above

Reply:
Hey @jnewc Originally I was unwrapping the PrimerValidationError wrongly, I presumed that we would get a `PrimerValidationError[]` array back but in actual fact we get an object with a `validationErrors ` field on it that contains the array.

Reply:
Resolved in latest changes!